test/unittest/CMakeLists.txt

```diff
@@ -9,20 +9,36 @@ include_directories(${SYCLBLAS_TEST})
 
 # compiling tests
 file(GLOB SYCL_UNITTEST_SRCS
-  ${SYCLBLAS_UNITTEST}/blas1_copy_test.cpp
-  ${SYCLBLAS_UNITTEST}/blas1_swap_test.cpp
-  ${SYCLBLAS_UNITTEST}/blas1_axpy_test.cpp
-  ${SYCLBLAS_UNITTEST}/blas1_scal_test.cpp
-  ${SYCLBLAS_UNITTEST}/blas1_asum_test.cpp
-  ${SYCLBLAS_UNITTEST}/blas1_dot_test.cpp
-  ${SYCLBLAS_UNITTEST}/blas1_nrm2_test.cpp
-  ${SYCLBLAS_UNITTEST}/blas1_rotg_test.cpp
-  ${SYCLBLAS_UNITTEST}/blas1_iamax_test.cpp
-  ${SYCLBLAS_UNITTEST}/blas1_iamin_test.cpp
-  ${SYCLBLAS_UNITTEST}/blas2_gemv_test.cpp
-  ${SYCLBLAS_UNITTEST}/blas2_ger_test.cpp
-  ${SYCLBLAS_UNITTEST}/blas3_gemm_test.cpp
-  ${SYCLBLAS_UNITTEST}/sycl_buffer_test.cpp
+  # Blas 1 tests
+  ${SYCLBLAS_UNITTEST}/blas1/blas1_copy_test.cpp
+  ${SYCLBLAS_UNITTEST}/blas1/blas1_swap_test.cpp
+  ${SYCLBLAS_UNITTEST}/blas1/blas1_axpy_test.cpp
+  ${SYCLBLAS_UNITTEST}/blas1/blas1_scal_test.cpp
+  ${SYCLBLAS_UNITTEST}/blas1/blas1_asum_test.cpp
+  ${SYCLBLAS_UNITTEST}/blas1/blas1_dot_test.cpp
+  ${SYCLBLAS_UNITTEST}/blas1/blas1_nrm2_test.cpp
+  ${SYCLBLAS_UNITTEST}/blas1/blas1_rotg_test.cpp
+  ${SYCLBLAS_UNITTEST}/blas1/blas1_iamax_test.cpp
+  ${SYCLBLAS_UNITTEST}/blas1/blas1_iamin_test.cpp
+
+  # Blas 2 tests
+  ${SYCLBLAS_UNITTEST}/blas2/blas2_gemv_test.cpp
+  ${SYCLBLAS_UNITTEST}/blas2/blas2_ger_test.cpp
+
+  # Blas 3 tests
+  # ${SYCLBLAS_UNITTEST}/blas3/blas3_gemm_test.cpp
```

Comment:
why is this commented?

Reply:
No reason - I just forgot to clean that bit up! I'll fix it now.